components/gcp/container_v2/launch.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -x 
+
+
+NAME=$(curl \
+ -X POST \
+ -H "Authorization: Bearer ${GCLOUD_AUTH_TOKEN}" \
+ -H "Content-Type: application/json" \
+  https://${ENDPOINT}/ui/projects/${PROJECT_ID}/locations/${REGION}/trainingPipelines \
+ -d $1 |  jq -r '.name')
+
+
+get() {
+	curl \
+	 -H "Authorization: Bearer ${GCLOUD_AUTH_TOKEN}" \
+	 -H "Content-Type: application/json" \
+	 https://${ENDPOINT}/v1/$NAME
+}
+
+STATE=$(get | jq -r '.state')
+while [ $STATE != "PIPELINE_STATE_SUCCEEDED" ]; do
```

Comment:
There are additional states to consider: https://github.com/googleapis/python-aiplatform/blob/master/google/cloud/aiplatform_v1/types/pipeline_state.py#L26

Reply:
thanks @sasha-gitg. Yeah this needs more proper handling. it's currently a poc stage and not ready for production.